test/Microsoft.ML.Tests/TrainerEstimators/FAFMEstimator.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.ML.Core.Data;
+using Microsoft.ML.Runtime.Data;
+using Microsoft.ML.Runtime.FactorizationMachine;
+using Microsoft.ML.Runtime.Learners;
+using Microsoft.ML.Runtime.RunTests;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.ML.Tests.TrainerEstimators
+{
+    public partial class TrainerEstimators : TestDataPipeBase
+    {
+        [Fact]
```

Comment:
> [Fact] [](start = 7, length = 7)

The test is failing to check whether the input is valid for fit. Resolve before check in.

Reply:
Thanks @Zruty0 for fixing the mismatch between the active and inactive columns. 

---
In reply to: [218592628](https://github.com/dotnet/machinelearning/pull/912#discussion_r218592628) [](ancestors = 218592628)